src/renderer/widgets/RemoveProxy/model/remove-proxy-model.ts

```diff
@@ -0,0 +1,209 @@
+import { UnsignedTransaction } from '@substrate/txwrapper-polkadot';
+import { combine, createEvent, createStore, sample } from 'effector';
+import { spread } from 'patronum';
+import { ApiPromise } from '@polkadot/api';
+
+import { Account, Chain, HexString, ProxyAccount, ProxyGroup } from '@shared/core';
+import { accountUtils, walletModel, walletUtils } from '@entities/wallet';
+import { dictionary } from '@shared/lib/utils';
+import { getSignatoryAccounts } from '@pages/Operations/common/utils';
+import { proxyModel } from '@entities/proxy';
+
+export const enum Step {
+  CONFIRMATION,
+  SIGNING,
+  SUBMIT,
+}
+
+type FlowStartedProps = {
+  chain: Chain;
+  proxyAccount: ProxyAccount;
+};
+const flowStarted = createEvent<FlowStartedProps>();
```

Comment:
what do you think about more meaningful `removeStarted`?

Reply:
Agree, we control the naming, no need for unnecessary abstraction